GitUI/UserControls/GPGKeys/GPGSecretKeysCombobox.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.ComponentModel;
+using System.Diagnostics;
+using GitCommands.Gpg;
+
+namespace GitUI.UserControls.GPGKeys
+{
+    public partial class GPGSecretKeysCombobox : GitModuleControl
```

Comment:
Can we avoid inheriting from `GitModuleControl`?

Reply:
How do you suggest I get the current module and the ui command source and all that?

Reply:
For example:
https://github.com/gitextensions/gitextensions/blob/99ece765b4a76a58b1d3160f914d9c6f0ab5581f/GitUI/CommandsDialogs/FormBrowse.cs#L266-L268
https://github.com/gitextensions/gitextensions/blob/99ece765b4a76a58b1d3160f914d9c6f0ab5581f/GitUI/CommandsDialogs/FormBrowse.cs#L298